.travis.yml

```diff
@@ -7,6 +10,11 @@ php:
   - 5.6
   - hhvm
 
+ # cache composer cache across builds
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
It might give an additional speed-up if you add `vendor` to the list so that already current vendor libraries don't have to be fetched from the composer cache. Just a minimal thing, but it probably won't hurt.